go.mod

```diff
@@ -1,10 +1,20 @@
 module github.com/lmatte7/meshtastic-go
 
-go 1.16
+go 1.23.0
+
+toolchain go1.23.4
 
 require (
-	github.com/golang/protobuf v1.5.0
+	github.com/golang/protobuf v1.5.4
 	github.com/lmatte7/gomesh v0.2.0
-	github.com/urfave/cli/v2 v2.3.0
-	google.golang.org/protobuf v1.26.0
-)
\ No newline at end of file
+	github.com/urfave/cli/v2 v2.27.6
+	google.golang.org/protobuf v1.36.6
+)
+
+require (
+	github.com/cpuguy83/go-md2man/v2 v2.0.6 // indirect
```

Comment:
It doesn't look like these are used anywhere, could you remove them? I'd also like to avoid imports when possible to help keep the project small and portable